jcl/src/jdk.jcmd/share/classes/openj9/tools/attach/diagnostics/tools/Util.java

```diff
@@ -139,4 +140,16 @@ static void getTargetInformation(AttachProvider theProvider, VirtualMachineDescr
 		}
 	}
 
+	/**
+	 * Print the Properties object in key-value format.
+	 * This differs from Properties.list in that is does not truncate long values.
```

Comment:
Just noticed another one. is -> it
also  Properties.list ->  Properties.list() would be better.